nemo_curator/image/classifiers/aesthetic.py

```diff
@@ -25,7 +26,7 @@
 # MLP code taken from LAION Aesthetic V2
 # https://github.com/christophschuhmann/improved-aesthetic-predictor/blob/main/simple_inference.py
 class MLP(nn.Module):
-    def __init__(self, input_size, xcol="emb", ycol="avg_rating"):
```

Comment:
Hopefully this doesn't change any behavior downstream

Reply:
Oh good catch, I don't know why that was changed. I can revert it back.